doctr/datasets/vocabs.py

```diff
@@ -66,6 +67,7 @@
         + VOCABS["danish"]
         + VOCABS["finnish"]
         + VOCABS["swedish"]
+        + VOCABS["hindi"]
```

Comment:
We should remove this for the moment because as you can see it contains only european characters and we don't have a dataset which includes hindi yet :)